app/controllers/predicates/AuthoriseAsAgentWithClient.scala

```diff
@@ -53,27 +54,28 @@ class AuthoriseAsAgentWithClient @Inject()(enrolmentsAuthService: EnrolmentsAuth
 
     request.session.get(SessionKeys.clientVRN) match {
       case Some(vrn) =>
-        logger.debug(s"[AuthoriseAsAgentWithClient][invokeBlock] - Client VRN from Session: $vrn")
+        debug(s"[AuthoriseAsAgentWithClient][invokeBlock] - Client VRN from Session: $vrn")
         enrolmentsAuthService.authorised(delegatedAuthRule(vrn)).retrieve(Retrievals.affinityGroup and Retrievals.allEnrolments) {
           case None ~ _ =>
+            errorLog("[][] - no enrolment found")
             Future.successful(serviceErrorHandler.showInternalServerError)
           case _ ~ allEnrolments =>
             val agent = Agent(allEnrolments)
             val user = User(vrn, active = true, Some(agent.arn))
             block(user)
         } recover {
           case _: NoActiveSession =>
-            logger.debug("[AuthoriseAsAgentWithClient][invokeBlock] - Agent does not have an active session, rendering Session Timeout")
+            errorLog("[AuthoriseAsAgentWithClient][invokeBlock] - Agent does not have an active session, rendering Session Timeout")
             Unauthorized(sessionTimeoutView())
           case _: AuthorisationException =>
-            logger.debug("[AuthoriseAsAgentWithClient][invokeBlock] - Agent does not have delegated authority for Client")
```

Comment:
warn

Reply:
why is this warn? its an authorisation error